### PR TITLE
[modbus.studer] Some little markdown and grammar corrections

### PR DESCRIPTION
**Pull request by thopiekar**
Nothing extra special, but some corrections to the docs.

### CODE REVIEW

File: bundles/org.openhab.binding.modbus.studer/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Note, that the things will show up under the Modbus binding.
 
 | Thing Type IDs | Description                                                                                                          | Picture                             |
 |----------------|----------------------------------------------------------------------------------------------------------------------|-------------------------------------|
-| bsp            | For BSP that offer a highly precise measuring for Xtender, VarioTrack and VarioString systems                        | ![BSP](doc/bsp.png)                 |
+| bsp            | For BSP that offers a highly precise measuring for Xtender, VarioTrack and VarioString systems                        | ![BSP](doc/bsp.png)                 |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.modbus.studer/README.md:16]:
```suggestion
| bsp            | For BSP that offers a highly precise measuring for Xtender, VarioTrack and VarioString systems                       | ![BSP](doc/bsp.png)                 |
```

File: bundles/org.openhab.binding.modbus.studer/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This extension adds support for the Studer protocol.
 
-Studer Innotec, founded in 1987 by Roland Studer, is an ISO certified company that develops and manufactures inverters, inverter/chargers and MPPT solar charge controllers to communicate over the Modbus protocol entirely in Switzerland
+Studer Innotec, founded in 1987 by Roland Studer, is an ISO-certified company that develops and manufactures inverters, inverter/chargers and MPPT solar charge controllers to communicate over the Modbus protocol entirely in Switzerland
```
Comment (lsiepel) [inline bundles/org.openhab.binding.modbus.studer/README.md:5]:
```suggestion
Studer Innotec, founded in 1987 by Roland Studer, is an ISO-certified company that develops and manufactures inverters, inverter/chargers and MPPT solar charge controllers to communicate over the Modbus protocol entirely in Switzerland.
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Two very minor details.

**Review by giovannimirulla — APPROVED** [review]
Thanks for your review @lsiepel

**Comment by lsiepel** [comment]
> Thanks for your review @lsiepel and @thopiekar

@giovannimirulla  marked both comments as resolved, but I see no commit from @thopiekar

**Review by lsiepel — APPROVED** [review]
LGTM

**Pull request merged**